app/views/devices_guidance/how_to_order_for_independent_special_schools.html.erb

```diff
@@ -0,0 +1,215 @@
+    </p>
+
+    <h3 class="govuk-heading-m">
+      2. Check what pupils need
+    </h3>
+    <p class="govuk-body">
+      We recommend contacting pupils, families or schools as soon as possible to find out the following information.
+    </p>
+
+    <h4 class="govuk-heading-s">
+      Laptops
+    </h4>
+    <p class="govuk-body">
+      To order laptops, you need to find out which:
+    </p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>
+        eligible pupils do not have access to a laptop or tablet
+      </li>
+      <li>
+        <%= govuk_link_to "Type of laptop and IT settings are most suitable for each care leaver", '/devices/device-specification' %>
+        - if they need Google Chromebooks, you should ask for their educational setting’s domain for
+        <span class="app-no-wrap">Google Workspace for Education Fundamentals</span>
+        <span class="app-no-wrap">(formerly G Suite for Education)</span>, and recovery email address. We’ll ask you for
+        these after you place your order.
+      </li>
+    </ul>
+    <p class="govuk-body">
+      <%= govuk_link_to 'Find out about safeguarding responsibilities, laptop ownership and providing IT support', '#step-6' %>.
+    </p>
+
+    <h4 class="govuk-heading-s">
+      Internet access
+    </h4>
+    <p class="govuk-body">
+      To request internet access, you need to find out which:
+    </p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>
+        eligible pupils do not have internet access at home
+      </li>
+      <li>
+        <%= govuk_link_to 'type of internet access is most suitable for each pupil', connectivity_home_path %>
+        (you also need to gather mobile phone information if you want to request extra data for pupils on participating networks)
+      </li>
+    </ul>
+    <p class="govuk-body">
+      You do not need to complete this step before moving on to step 3.
+    </p>
+
+    <h3 class="govuk-heading-m">
+      3. Complete and submit your contact details form
+    </h3>
+    <p class="govuk-body">
+      We need some information from you to set up your Get help with technology service account, even if you’ve used
+      the service before. You’ll place your orders using this account.
+    </p>
+    <p class="govuk-body">
+      If you’ve been nominated to order for your LA, you’ll receive an email with a link to an online form.
+      You need to use that form to tell us:
+    </p>
+    <ol class="govuk-list govuk-list--number">
+      <li>
+        Your full name.
+      </li>
+      <li>
+        Your local authority name and 3-digit code.
+      </li>
+      <li>
+        The delivery address we should send your laptops to (all devices will be delivered to the same location at your local authority).
+      </li>
+      <li>
+        The person we should contact when we deliver your laptops.
+      </li>
+    </ol>
```

Comment:
On the prototype, this was an order list. My question is, should it be an unordered list? There isn't specific importance to any of these and its not that they are required in this specific order. They are just a list of things we need them to tell us.

Reply:
Change to:

- your full name
- your local authority’s name and 3-digit code
- the delivery address we should send your laptops to (all devices will be delivered to the same location at your local authority)
- the person we should contact when we deliver your laptops